migrations/versions/83a221435ceb_add_business_in_regulatory_alert.py

```diff
@@ -0,0 +1,44 @@
+"""add business in regulatory alert
+
+Revision ID: 83a221435ceb
+Revises: bd643a8d5269
+Create Date: 2024-11-06 14:01:58.678276
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+from app.domain.regulations import get_default_business
+
+# revision identifiers, used by Alembic.
+revision = "83a221435ceb"
+down_revision = "bd643a8d5269"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "regulatory_alert",
+        sa.Column("business_id", sa.Integer(), nullable=True),
+    )
+
+    default_buiness = get_default_business()
```

Comment:
:green_circle: Typo mais comme ce n'est que dans la migration, ce n'est sans doute pas bien grave :shrug: 

```suggestion
    default_business = get_default_business()
```

Reply:
ah oui bien vu !